docs/detections/rules-ui-create.asciidoc

```diff
@@ -558,11 +563,15 @@ used as an identifier across systems
 * `{{rule.spaceId}}`: Space ID of the rule
 * `{{rule.tags}}`: Tags of the rule
 * `{{rule.type}}`: Type of rule
-* `{{state.signals_count}}`: Number of alerts detected
 
-NOTE: The `{{context.rule.severity}}` and `{{context.rule.risk_score}}`
-placeholders contain the rule's default values even when the *Severity override*
-and *Risk score override* options are used.
+If the rule’s action frequency is not a summary of alerts (per each alert), it passes the following variables:
+
+* `{{alert.actionGroup}}`: Action group of the alert that scheduled actions for the rule
+* `{{alert.actionGroupName}}`: Human readable name of the action group of the alert that scheduled actions for the rule
+* `{{alert.actionSubgroup}}`: Action subgroup of the alert that scheduled actions for the rule
+* `{{alert.id}}`: ID of the alert that scheduled actions for the rule
+* `{{alert.flapping}}: A flag on the alert that indicates whether the alert status is changing repeatedly.
+* `{{state.signals_count}}`: Number of alerts detected
```

Comment:
`{{state.signals_count}}` variable is available in both cases: `summary of alerts` and `for each alert`